.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,8 +1,11 @@
-  ---
-name: "Bug Report"
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
 labels: bug
-assignee: kaikai1024
-  ---
+assignees: kaikai1024
```

Comment:
`assignees` is necessary? This report just record problem is enough. Assign to who, is another question later.

Reply:
It's friendly.

>  Assign to who, is another question later.